docs/DeveloperGuide.md

```diff
@@ -412,12 +412,12 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
 
 (For all use cases below, the **System** is the `TutorEase` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use Case: UC01 - Add student contact**
+**Use Case: UC01 - Add contact**
```

Comment:
Since we can add and manage a guardian contact as well, I have updated the user case to be more general.

Reply:
Good job identifying this! I agree that making it general is better! 👍